src/schema/v2/artist/index.ts

```diff
@@ -868,6 +868,38 @@ export const ArtistType = new GraphQLObjectType<any, ResolverContext>({
         resolve: ({ id }, options, { partnerArtistsForArtistLoader }) =>
           partnerArtistsForArtistLoader(id, options),
       },
+      partnerBiographyBlurb: {
+        description: "The Partner's provided biography for the artist",
+        args: {
+          ...markdown().args,
+        },
+        type: new GraphQLObjectType<any, ResolverContext>({
+          name: "partnerBiographyBlurb",
+          fields: {
+            text: {
+              type: GraphQLString,
+              resolve: ({ text }) => text,
+            },
+          },
+        }),
+        resolve: async (
+          { id },
+          { format },
+          { partnerArtistsForArtistLoader }
+        ) => {
+          const partnerArtists = await partnerArtistsForArtistLoader(id, {
+            size: 1,
+          })
+
+          if (partnerArtists && partnerArtists.length) {
+            const { biography } = first(partnerArtists) as any
```

Comment:
Doesn't this mean we might end up querying the bio defined by one partner and displaying it on an artwork that belongs to another partner?

It looks like we're selecting the first partner artist from the list of all partner artists associated with one Artsy.

Example:
If both `Gagosian` and `A+B Gallery` have `Patrick Hughes` listed as a partner artist and Gagosian was the one to register him first, following the logic here, I'd expect that Gagosian custom bio would be displayed for any Patrick Hughes private artworks A+B Gallery uploaded as well.

Reply:
I tested the above with `Mike Mandel`.

I have `Mike Mandel` registered as a partner artist for `Robert Mann Gallery` and `Commerce Test Partner`

I registered different versions of custom bio for each gallery
![Screenshot 2024-04-25 at 14 09 47](https://github.com/artsy/metaphysics/assets/8002618/d93d68ea-5004-458c-8bc4-b200d572489d)

![Screenshot 2024-04-25 at 14 10 38](https://github.com/artsy/metaphysics/assets/8002618/527ee738-d5de-4bce-ae04-9d385116a97f)

Now even if I access an [artwork](https://staging.artsy.net/artwork/639cad86592ddf000bfbed88) that belongs to `Commerce Test Partner`, I see the bio that was added by `Robert Mann Gallery`
![Screenshot 2024-04-25 at 14 12 32](https://github.com/artsy/metaphysics/assets/8002618/24d2fc9d-7eb4-47f5-903b-c0d0e2d2e8b8)

Reply:
Great catch @lidimayra , I think this is also the behavior on the existing 'biographyBlurb' on artist 😬 I wonder if that is OK

Reply:
I tested it here, it _seems_ fine (not sure, we might want to investigate deeper)
I also added a custom bio for Mike Mandel on Abakus Project gallery

I see their custom bio working fine on [the artist page](https://staging.artsy.net/partner/abakus-projects/artists/mike-mandel)

And the public artwork page uses [the Artsy provided bio](https://staging.artsy.net/artwork/mike-mandel-stacking-pampers)